gradle.properties

```diff
@@ -1,5 +1,5 @@
 # Magic options for more perf
-org.gradle.jvmargs=-Xmx2048M -XX:MaxMetaspaceSize=512m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8 -XX:+UseParallelGC
+org.gradle.jvmargs=-Xmx3048M -XX:MaxMetaspaceSize=512m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8 -XX:+UseParallelGC
```

Comment:
3072

But is this increase even needed?

Reply:
I was getting OOM exceptions while building -- it's up to you on whether to include this but I will need to go back and change it locally every time